draft-lenders-core-coap-dtls-svcb.md

```diff
@@ -60,61 +61,52 @@ normative:
 informative:
   RFC8323: coap-tcp
   I-D.ietf-core-dns-over-coap: doc
+  RFC4944: 6lo
 
 --- abstract
 
-This document specifies the usage of Service Parameters as used in SVCB ("Service Binding") DNS resource
-records for the discovery of transport-layer-secured CoAP services.
+This document specifies an Application-Layer Protocol Negotiation (ALPN) ID for
+transport-layer-secured CoAP services.
 
 --- middle
 
 # Introduction
 
-{{-svcb}} specifies the "SVCB" ("Service Binding") DNS resource records for looking up
-communication endpoints of a service. Service Parameters (SvcParams) are used to
-carry that information.
-This document specifies which information from SvcParams can be used with CoAP services that are
-secured by transport security, namely TLS and DTLS.
-As an example, this information can be obtained as part of the discovery of DNS over CoAP (DoC) servers (see
-{{-doc}}) that deploy TLS or DTLS to secure their messages.
+Application-Layer Protocol Negotiation (ALPN) allows agreeing on an application-layer protocol during a Transport Layer Security (TLS) handshake.
+For that an ALPN ID is used.
+Furthermore, the ALPN ID can be used to discover Service Bindings (SVCB) via DNS, using SVCB resource records with the "alpn" Service Parameter Keys.
+As an example, this information can be obtained as part of the discovery of DNS over CoAP (DoC) servers (see {{-doc}}) that deploy TLS or DTLS to secure their messages.
+This document specifies ALPN IDs for CoAP services that are secured by transport security, namely TLS and DTLS.
 
 # Terminology
 
-SvcParams denotes the field in either DNS SVCB/HTTPS records as defined in {{-svcb}}, or DHCP and RA
-messages as defined in {{-dnr}}.
-
 {::boilerplate bcp14-tagged}
```

Comment:
We don't use the bcp14 identifiers at all in the document.

Reply:
Anything we should add to the Terminology section otherwise? If not, I would just drop it completely.

Reply:
I'm not aware of anything that warrants a terminology section.